runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -8299,9 +8299,11 @@ class INTERPRETER_CLASS
 	{
 		VM_BytecodeAction rc = GOTO_RUN_METHOD;
 		bool fromJIT = J9_ARE_ANY_BITS_SET(_currentThread->jitStackFrameFlags, J9_SSF_JIT_NATIVE_TRANSITION_FRAME);
+		bool fromNF = J9_ARE_ANY_BITS_SET(_currentThread->privateFlags2, J9_PRIVATE_FLAGS2_NF_INVOKE_BASIC);
```

Comment:
what does NF stand for?

Reply:
NamedFunction, it is the wrapper used to hold a `invokeBasic` MemberName